### PR TITLE
Cleanup Flatpak setup in The Basic Setup section

### LINKED ISSUE #214: Cleanup Flatpak setup in The Basic Setup section
### Problem

The current docs assume the user is running as far back as OS6 and might need to install Flatpak and add the AppCenter remote to it

### Proposal

Remove this section as it is all built-in with OS8. If GitBook somehow support allowing a dev to pick an older eOS version and view docs for that, I would be fine with that too.

### Prior Art (Optional)

_No response_

### PR DESCRIPTION
**Pull request by colinkiama**
Flatpak setup in Basic Setup section allows readers to pick instructions based on their OS version

Fixes #214

### CODE REVIEW

File: writing-apps/the-basic-setup.md [hunk 1]
```diff
@@ -90,10 +90,23 @@ sudo apt install elementary-sdk
...
+    {% tab title="OS 8 and Above" %}
+
+    {% endtab %}
+
+    {% tab title="OS 6 and Above" %}
+        ```bash
+        flatpak install -y appcenter io.elementary.Sdk
+        ```
+    {% endtab %}
+{% tab title="Before OS 6" %}
```
Comment (danirabbit) [inline writing-apps/the-basic-setup.md:103]: Anything before OS 6 is now EOL so we can just drop those instructions completely or you could make a tab for `elementary OS` and `Other Linux OS`

File: writing-apps/the-basic-setup.md [hunk 2]
```diff
@@ -90,10 +90,23 @@ sudo apt install elementary-sdk
...
 On elementary OS 6 beta and newer, you should already have the required Flatpak remote and platform pre-installed. On earlier versions or other OSes, you can add the remote and install the Flatpak platform and SDK:
 
-```bash
-flatpak remote-add --if-not-exists --system appcenter https://flatpak.elementary.io/repo.flatpakrepo
-flatpak install -y appcenter io.elementary.Platform io.elementary.Sdk
-```
+{% tabs %}
+    {% tab title="OS 8 and Above" %}
+
+    {% endtab %}
```
Comment (danirabbit) [inline writing-apps/the-basic-setup.md:96]: Sdk has to be installed manually even on OS 8. Only the platform is pre-installed

File: writing-apps/the-basic-setup.md (hunk 1 above)
Comment (colinkiama) [reply writing-apps/the-basic-setup.md]: I went with the second option :)

File: writing-apps/the-basic-setup.md (hunk 2 above)
Comment (colinkiama) [reply writing-apps/the-basic-setup.md]: SDK install instructions have been added now

**Review by danirabbit — APPROVED** [review]
LGTM! Nice job :)

**Comment by colinkiama** [comment]
Uh oh, there's an issue with the formatting. I think that the indenting before the bash code blocks needs to be removed but I can't preview the changes:
Screenshot from 2024-10-10 21 46 23

**Comment by danirabbit** [comment]
@colinkiama should be fixed now!

**Pull request merged**